content/en/blog/posts/2022-02-01-build-release-v0.8.0/index.md

```diff
@@ -0,0 +1,116 @@
+
+**Note**: For more details, please see the [docs](https://github.com/shipwright-io/build/blob/main/docs/buildrun.md#understanding-failed-buildruns).
+
+### Local Source Upload
+
+At Shipwright, we've spent a lot of time trying to figure out the best ways to simplify the experience when building container images. In this release we are introducing a new feature that dramatically improves it, we call it **Local Source Upload** .
+
+This feature allows users to build container images from their local source code, improving the developer experience and moving them closer to the inner dev loop(_single developer workflow_).
+
+![shp cli](image002.png)
+_The photo above illustrates the use of the new shp cli subcommand._
+
+Go ahead and give it a try! The feature is now available in the [v0.8.0](https://github.com/shipwright-io/cli/releases/tag/v0.8.0) cli, look for `shp build upload`!
+
+## Important Notes
+
+{{< alert type="success" title="Action Required" >}}If you previously used the go types with parameters, then you will need to make slight changes to adopt to the changed type structure.{{< /alert >}}
+
+## Installing Shipwright
+
+### Build
+
+1. Install Tekton v0.30.0
+
+   ```bash
+   kubectl apply -f https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.30.0/release.yaml
+   ```
+
+2. Install v0.8.0 using the release YAML manifest:
+
+   ```bash
+   kubectl apply -f https://github.com/shipwright-io/build/releases/download/v0.8.0/release.yaml
+   ```
+
+3. (Optionally) Install the sample build strategies using the YAML manifest:
+
+   ```bash
+   kubectl apply -f https://github.com/shipwright-io/build/releases/download/v0.8.0/sample-strategies.yaml
+   ```
+
+### SHP CLI
+
+#### Windows
+
+```sh
+curl --silent --fail --location https://github.com/shipwright-io/cli/releases/download/v0.8.0/cli_0.8.0_windows_x86_64.tar.gz | tar xzf - shp.exe
+shp help
+```
+
+#### Mac
+
+```sh
+curl --silent --fail --location https://github.com/shipwright-io/cli/releases/download/v0.8.0/cli_0.8.0_macOS_x86_64.tar.gz | tar -xzf - -C /usr/local/bin shp
+shp help
+```
+
+#### Linux
+
+```sh
+curl --silent --fail --location "https://github.com/shipwright-io/cli/releases/download/v0.8.0/cli_0.9.0_linux_$(uname -m | sed 's/aarch64/arm64/').tar.gz" | sudo tar -xzf - -C /usr/bin shp
+shp help
+```
+
+### Operator
+
+To deploy and manage Shipwright Builds in your cluster, first make sure the operator is installed and running on your cluster. You can follow the instructions on [OperatorHub](https://operatorhub.io/operator/shipwright-operator).
+
+Next, create the following:
+
+```yaml
+---
+apiVersion: operator.shipwright.io/v1alpha1
+kind: ShipwrightBuild
+metadata:
+  name: shipwright-operator
+spec:
+  targetNamespace: shipwright-build
+```
+
+The operator will deploy Shipwright Builds in the provided targetNamespace. When `.spec.targetNamespace` is not set, the namespace will default to shipwright-build.
```

Comment:
This actually can be dropped. This was fixed in v0.7.0 (at least), but we haven't updated our docs yet. See https://github.com/shipwright-io/operator/issues/51